Dockerfile

```diff
@@ -16,11 +17,21 @@ make && \
 make install && \
 make install-etc && \
 make install-dev && \
+# install skypeweb
 cd && \
 tar zxvf 1.4.tar.gz && \
 cd skype4pidgin-1.4/skypeweb && \
 make && \
 make install && \
+# skypeweb end
+# install telegram purple
+cd && \
+tar zxvf telegram-purple_1.3.1.orig.tar.gz && \
+cd telegram-purple && \
+./configure && \
+make && \
+make install && \
+# telegram end
```

Comment:
please remove this line and add a line to delimit the next install phase ("install bitlbee facebook")